common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -267,9 +271,6 @@ def create_response_element(self, **kwargs):
 
         *answer_attr*: The "answer" attribute on the tag itself (treated as an
         alias to "expect", though "expect" takes priority if both are given)
-
-        *group_label*: Text to represent group of inputs when there are
```

Comment:
Actually, I need to follow up with @muhammad-ammar and/or @muzaffaryousaf about this. I saw in the code that "group_label" does exist in code, so I need to understand the interaction between it and question text. This test might need to be changed again to restore coverage of group_label.